Translation corrections (#463)
07th-mod/higurashi-console-arcs: Update/omot_015.txt

			NULL, " Did you seriously think I didn't know about this?", Line_WaitForInput);
	ModPlayVoiceLS(4, 15, "ps3/s14/15/2115000078", 256, TRUE);
	OutputLine(NULL, "　せめてもの温情で職だけは持たせてやる。",

> 			NULL, " The very least kindness I can impart on you is getting you a new job.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " Out of kindness, I'll at least let you keep your job.", Line_WaitForInput);
```
